recipes/cfn-flip/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "cfn-flip" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/awslabs/aws-cfn-template-flip/archive/{{ version }}.tar.gz
+  sha256: 2ea2d115a69416acd577c2d97c7731d1a8fc313544831106e756ef22995c73a1
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - cfn-flip=cfn_flip.main:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - click
+    - pip
+    - python
+    - pyyaml >=4.1
+    - six
+  run:
+    - click
+    - python
+    - pyyaml >=4.1
+    - six
+
+test:
+  imports:
+    - cfn_clean
+    - cfn_flip
+    - cfn_tools
+  commands:
+    - cfn-flip --help
+  requires:
+    - pytest >=4.3.0
+    - pytest-cov
+    - pytest-runner
+
+about:
+  home: https://github.com/awslabs/aws-cfn-template-flip
+  license: Apache2
```

Could you please change it to
```suggestion
  license: Apache-2.0
```